python practice/day3/solution.py

```diff
@@ -1,6 +1,19 @@
-n=int(input())
-m=n % 2
-if ((m!=0) or (n>=6 and n<=20)):
+##n=int(input())
```

Comment:
Extra comments cause code seems dirty.
Please remove extra comment and push solution again.